src/css/animations.scss

```diff
@@ -1,19 +1,13 @@
-@each $animName in ("changeAnimEven", "changeAnimOdd") {
-    @keyframes #{$animName} {
-        0% {
-            transform: scale(1, 1);
-        }
-
-        50% {
-            transform: scale(1.03, 1.03);
-        }
+@include MakeAnimationWrappedEvenOdd(0.2s ease-in-out, "changeAnim") {
+    0% {
+        transform: scale(1, 1);
+    }
 
-        100% {
-            transform: scale(1, 1);
-        }
+    50% {
+        transform: scale(1.03, 1.03);
     }
 
-    .#{$animName} {
```

Comment:
why did you remove it here?

Reply:
I think that was from an older version of shapez.io

Reply:
Fixed back to normal

Reply:
But missing spaces

Reply:
Fully fixed